pandas/tests/io/test_parquet.py

```diff
@@ -712,6 +727,13 @@ def test_timestamp_nanoseconds(self, pa):
         df = pd.DataFrame({"a": pd.date_range("2017-01-01", freq="1n", periods=10)})
         check_round_trip(df, pa, write_kwargs={"version": "2.0"})
 
+    @td.skip_if_no("pyarrow", min_version="0.14")
+    def test_timezone_aware_index(self, pa, timezone_aware_date_list):
+        idx = 5 * [timezone_aware_date_list]
+
+        df = pd.DataFrame(index=idx)
+        check_round_trip(df, pa, write_kwargs={"version": "2.0"})
```

Comment:
Is the `write_kwargs={"version": "2.0"}` needed here? (in the previous test this was added because that was explicitly testing the conversion with this kwarg, but the other tests generally don't specify this I think)